src/routes/NonAuthLayout/Login/Login.connector.js

```diff
@@ -6,9 +7,10 @@ import { getReturnToURL, resetReturnToURL } from 'router/AuthRoute/AuthRoute.sto
 import mobileRedirect from 'util/mobileRedirect'
 
 export function mapStateToProps (state, props) {
+  const queryParams = qs.parse(props.location.search.substring(1))
```

Comment:
I believe `getQuerystringParam` would be appropriate and simplifying here too, i.e.:

`const returnToUrl = getQuerystringParam('returnToUrl') || getReturnToUrl(state)`